testplugins/src/main/java/org/spongepowered/test/FoodChangeTest.java

```diff
@@ -0,0 +1,49 @@
+package org.spongepowered.test;
+
+import static org.spongepowered.api.data.DataTransactionResult.DataCategory.REPLACED;
+import static org.spongepowered.api.data.DataTransactionResult.DataCategory.SUCCESSFUL;
+
+import org.spongepowered.api.data.DataTransactionResult;
+import org.spongepowered.api.data.key.Keys;
+import org.spongepowered.api.data.value.immutable.ImmutableValue;
+import org.spongepowered.api.data.value.mutable.MutableBoundedValue;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.Order;
+import org.spongepowered.api.event.data.ChangeDataHolderEvent;
+import org.spongepowered.api.event.filter.cause.First;
+import org.spongepowered.api.event.filter.data.GetKey;
+import org.spongepowered.api.plugin.Plugin;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.channel.MessageChannel;
+
+import java.util.Optional;
+
+@Plugin(id = "food-change-test", authors = "Aaron1011")
+public class FoodChangeTest {
```

Comment:
If this is going to constantly print to the player, should make it a toggle to join in (reduces the amount of commenting when debugging implementation).